### ISSUE TITLE
Can we have the option to turn off in-game music but turn it on in everywhere else?

### ISSUE THREAD

**Issue by Bedtimestory9** [ValveSoftware/Dota2-Gameplay#18709]
### Description

I like music, but having it repeatedly on while I'm spectating is not very pleasant. Or in-game battle sound repeatedly speeding up my heart rate.

I just want it to be off while I'm spectating and in game, but turn it on in client Ui, draft, etc. Thank you.

### Example Match ID (and possibly Timestamp)

Nomatch

**Comment by christy2951hernandez** [comment]
Absolutely, having separate volume controls for in-game music and other audio elements is a common request in many games.
Players can personalize their experience based on preference. Some might enjoy music during battles, while others prefer a more focused soundscape. Repetitive music during gameplay can become tiring. Turning it off allows players to focus on sound cues and in-game communication. Spectating matches can be more enjoyable without intrusive music.
